package.json

```diff
@@ -6,7 +6,8 @@
   "scripts": {
     "test": "echo \"Error: no test specified\" && exit 1",
     "build": "webpack",
-    "watch": "webpack --watch"
+    "watch": "webpack --watch",
+    "server": "webpack-dev-server"
```

Comment:
**OPTIONAL**
Let's change `"server": "webpack-dev-server"` to `"start": "webpack-dev-server"` to get rid of an error raised on an attempt to start the server.